src/TorchSharp/Optimizers/Rprop.cs

```diff
@@ -3,9 +3,15 @@
 using System.Collections.Generic;
 using System.Linq;
 using static TorchSharp.torch;
+using static TorchSharp.torchvision;
+using static TorchSharp.Utils.LEB128Codec;
 
 namespace TorchSharp
 {
+    using System.IO;
+    using System.Net;
+    using Google.Protobuf.WellKnownTypes;
+    using ICSharpCode.SharpZipLib.Core;
```

Comment:
why we don't add these to the top?

Reply:
Something VS added. No idea why not at the top. I'll fix.